packages/xchain-aggregator/package.json

```
@@ -29,7 +29,7 @@
     "directory": "release/package"
   },
   "dependencies": {
-    "@chainflip/sdk": "1.7.0",
+    "@chainflip/sdk": "1.9.2",
```

```suggestion
    "@chainflip/sdk": "^1.9.2",
```
As a library, shouldn't xchainjs always use caret version dependencies in general? That way the application has final control over the package version, without resorting to overriding it with `"resolutions"` or waiting for xchainjs to update.